client/src/app/components/shared/signin-form/signin-form.component.ts

```diff
@@ -33,20 +39,64 @@ export class SigninFormComponent implements OnInit{
 
   submit() {
     if(this.urlState === '/login/signin') {
-      this.authData.forEach((item:any) => {
-        if (item !== null && item !== undefined) {
-          if (item.email === this.signInForm.get('email')?.value) {
-            if (item.password === this.signInForm.get('password')?.value) {
-              this.cookieService.createUser(this.authData.email);
-              this.router.navigate(['/dashboard']);
+      this.credentialService.getCredentials().subscribe(res => {
```

Comment:
add a condition to check if users are exsists in database before login. users table if null, this condition will create a default user in database